docs/conversion.md

```diff
@@ -15,7 +15,7 @@ This document outlines all the conversion details regarding `docker-compose.yaml
 | depends_on |  | N |  |  |
 | dns |  | N |  |  |
 | dns_search |  | N |  |  |
-| tmpfs |  | N |  |  |
+| tmpfs |  | Y | [Pod.Spec.Containers.Volumes.EmptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) | Creates `emptyDir` volume with `medium:Memory` & mounts given directory inside container | 
```

Comment:
Still nitpicking this, but could be better:
`Creates `emptyDir` volume with `medium` set to `memory`. `medium:Memory` is a tad confusing